### PR TITLE
Feature/full calendar

### PR DESCRIPTION
**Pull request by kevinhartmann23**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- Adds calendar functionality -- DAY/WEEK/MONTH VIEW
- DUMMY data for events (replace with API calls) - annotations included
- initial styling for calendar 

## *Instructions for reviewer:*
*Where should the reviewer start?*
*How should this be manually tested?*
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,12 +3,19 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.1.5",
+    "@emotion/styled": "^11.3.0",
```
Comment (DrewBradley) [inline package.json:7]: This project has been full of emotions.

File: src/components/calendar/Calendar.js [hunk 2]
```diff
@@ -1,11 +1,77 @@
...
+import FullCalendar from '@fullcalendar/react'
+import dayGridPlugin from '@fullcalendar/daygrid'
+import timeGridPlugin from '@fullcalendar/timegrid'
+import interactionPlugin from "@fullcalendar/interaction"
+import { INITIAL_EVENTS, createEventId } from './event-utils'
+import {CalendarWrapper} from './CalendarWrapper'
+// import './Calendar.css';
+
 
 const Calendar = () => {
```
Comment (DrewBradley) [inline src/components/calendar/Calendar.js:11]: Beautiful work! I love this package!

File: src/components/calendar/CalendarWrapper.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import styled from "@emotion/styled";
```
Comment (DrewBradley) [inline src/components/calendar/CalendarWrapper.js:1]: Great job on this styled component!

File: src/components/calendar/event-utils.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+let eventGuid = 0
+const today = new Date()
+let todayStr = today.toLocaleDateString("pt-br").split('/').reverse().join('-')
```
Comment (kristenmb) [inline src/components/calendar/event-utils.js:3]: Great work figuring out this formatting. Thanks for muscling through that!

File: src/components/calendar/event-utils.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  {
+    id: createEventId(),
+    title: 'All-day event',
+    start: todayStr
+  },
+  {
+    id: createEventId(),
+    title: ' - Timed event',
+    start: todayStr +  insertTimeHere
+  }
```
Comment (DrewBradley) [inline src/components/calendar/event-utils.js:24]: These are events!

**Review by DrewBradley — APPROVED** [review]
Kevin, you are a genius!

File: src/components/calendar/CalendarWrapper.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+export const CalendarWrapper = styled.div`
```
Comment (kristenmb) [inline src/components/calendar/CalendarWrapper.js:3]: This is such a cool addition. So odd that we had to implement styling this way but seems cool!

File: src/components/calendar/Calendar.js [hunk 7]
```diff
@@ -1,11 +1,77 @@
...
+  const renderEventContent = (clickInfo) => {
+    return (
+      <>
+        <b>{clickInfo.timeText}</b>
+        <i>{clickInfo.event.title}</i>
+      </>
+    )
+  }
+
+  const handleEventClick = (clickInfo) => {
```
Comment (kristenmb) [inline src/components/calendar/Calendar.js:24]: Maybe this is where we trigger the Form component

**Review by kristenmb — APPROVED** [review]

**Review by richardltyler — APPROVED** [review]
Looks great!

**Pull request merged**